setup.py

```diff
@@ -16,5 +16,8 @@
     include_package_data=True,
     license=package['license'],
     description=package['description'] if 'description' in package else package_name,
-    install_requires=[]
+    install_requires=[
```

Comment:
We might want to specify the version of `dash` we use here! 

Also, where do we need `dash-html-components`? I don't think it's used anywhere in `dash-bio` itself, just in the demo apps. 

Might also be a good idea to go through the pure Python components (clustergram/manhattan plot/volcano plot) to see what their requirements are, and cross-reference with `requirements.txt` to get the exact versions. What do you think?

Reply:
I think we should keep those separate, since we're just providing a library of components and not an app-building library.

>For the dash version, what version should be the minimum?

Let's go with `0.36.0`, since that represents a big change (removal of events).

Reply:
Oops, just saw @alexcjohnson's reply. Let's do that instead!

Reply:
@shammamah So you're suggesting specifying 'dash>=0.36.0'? I guess, at this point, we might as well wait for the release @alexcjohnson mentioned, and then we'll specify 'dash>=[that release version]' and remove 'dash-html-components'...

Regarding cross-referencing dependencies in `requirements.txt`, I'm not in favour of it; I have reread https://packaging.python.org/discussions/install-requires-vs-requirements/, I have [quoted](https://github.com/plotly/dash-bio/issues/135#issuecomment-458467753) it before, and `requirements.txt` fit a different purpose.